v2/samples/containerservice/v1api20230202preview/refs/v1api20210701_workspace.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: machinelearningservices.azure.com/v1api20210701
+kind: Workspace
+metadata:
+  name: sampleworkspaces
+  namespace: default
+spec:
+  location: westus3
+  owner:
+    name: aso-sample-rg
+  sku:
+    name: Standard_S1
+    tier: Basic
+  allowPublicAccessWhenBehindVnet: false
+  identity:
+    type: SystemAssigned
+  storageAccountReference:
+    group: "storage.azure.com"
```

Comment:
minor: I just realized we're doing this here and it might also happen elsewhere, but AFAIK you don't need the quites around these `group`, `kind`, etc strings. In fact we don't use quotes above for any of the strings so it looks a bit funky that these have quotes when the rest don't.

IMO we should drop all of the quotes as I believe they are unneeded.

Reply:
You're right. I've removed the quotes from samples touched by this PR, but we have others to clean up as well.